src/python/pants/backend/python/lint/pylint/plugin_target_type.py

```diff
@@ -58,7 +58,7 @@ class PylintSourcePlugin(Target):
 
     alias = "pylint_source_plugin"
     core_fields = (
-        *(FrozenOrderedSet(COMMON_PYTHON_FIELDS) - {Dependencies}),
+        *(FrozenOrderedSet(COMMON_PYTHON_FIELDS) - {Dependencies}),  # type: ignore[misc]
```

Comment:
MyPy complains about mixing an `AsyncField` with a `PrimitiveField`, from what I grok from the error message. This seems very much like a bug in MyPy. If/when they fix it, MyPy will complain about the unnecessary ignore.